src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -159,12 +159,16 @@ private function configBuilder(string $namespace): ConfigBuilderInterface
 
         // If it does not start with Symfony\Config\ we dont know how to handle this
         if ('Symfony\\Config\\' !== substr($namespace, 0, 15)) {
-            throw new InvalidargumentException(sprintf('Could not find or generate class "%s".', $namespace));
+            throw new InvalidArgumentException(sprintf('Could not find or generate class "%s".', $namespace));
         }
 
         // Try to get the extension alias
         $alias = Container::underscore(substr($namespace, 15, -6));
 
+        if (false !== strpos($alias, '\\')) {
+            throw new InvalidArgumentException(sprintf('Nested bundle config class "%s" could not be handled.', $namespace));
```

Comment:
Technically, we are not talking about "bundles" but "extensions". 

Could we be more descriptive here?

Something more similar to this?
```
Could not find or generate class "%s". Make sure to use the "root" ConfigBuilder class. Ie use "Symfony\Config\FrameworkConfig" and not Symfony\Config\Framework\CacheConfig"
```

Reply:
I have tried to make message more descriptive, but without concrete classes.

Reply:
Oh. I missed this comment. 

I do agree that not using `Symfony\Config\Framework\CacheConfig` may be better, but it is also very hard to understand what "nested classes" really mean. Im not sure that my suggestion is the best.